dallinger/experiment_server/experiment_server.py

```diff
@@ -319,7 +319,10 @@ def advertisement():
         status = part.status
     except exc.SQLAlchemyError:
         status = None
-
+    try:
+        app_id = config.get('id')
```

Comment:
The following comment is not really about this pull request, but since I haven't been sure where to bring it up...I think that the support for specifying defaults this way is a mistake because it decentralizes where the defaults are specified. Two different lines of code can access the same setting with a different default. I think the configuration system would be more consistent if we change it so defaults are specified once when the setting is registered.